data/users.yml

```
@@ -1174,6 +1174,10 @@
   image: zipbooks.png
   alt: ZipBooks
   use: "Used for all our web apps and internal tools"
+-url: https://linkfire.com
```

```suggestion
- url: https://linkfire.com
```

👋 I think that will fix the build.

Thanks for the PR and for submitting to the Airtable!

---

All updated :)